editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/FuseIntegrationProjectCreatorRunnable.java

```diff
@@ -111,6 +111,14 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 			}
 		}
 
+		try {
+			Display.getDefault().syncExec( () -> postProjectCreationSetup(prj, subMonitor));
+		} finally {
+			subMonitor.done();
```

Comment:
no need to call done on submonitor see 4.0 https://eclipse.org/articles/Article-Progress-Monitors/article.html